src/integration-tests/tests/context/stress.rs

```diff
@@ -0,0 +1,333 @@
+#![allow(async_fn_in_trait)]
```

Comment:
Is this still necessary?

Reply:
Yes, without it clippy warns about the trait with async methods.